File: ruby-quest05-dev/ruby-quest05/ex01/my_map_mult_two.rb
##
##
## QWASAR.IO -- my_map_mult_two
##
##
## @param {Integer[]} param_1
##
## @return {integer[]}
##


def my_map_mult_two(param_1)
    i = 0
    result = [] # Create an empty array to store the updated values
  
    while i < param_1.length
      result << param_1[i] * 2 # Multiply each element by 2 and add it to the result array
      i += 1
    end
  
    return result # Return the updated array
  end

  #print my_map_mult_two([0, 1, 2, 3, 4, 5])
  
  

File: js-quest07-dev/js-quest07/ex00/my_map_mult_two.js
/**
 * 
 * #1 Program input : An integer array.
 * 
 * #2 Program output : Modified array(arr[] * 2).
 * 
 * Program iterate over array and multiply element by 2. 
 * 
 */

 function my_map_mult_two(param_1) {
     for(i = 0; i < param_1.length; i++){
        param_1[i] = param_1[i] * 2;
     }
        return param_1;
}


 //console.log(my_map_mult_two([1, 2, 3, 4, 5]));